packages/office-ui-fabric-react/src/components/List/List.tsx

```diff
@@ -400,14 +390,37 @@ export class List<T = any> extends React.Component<IListProps<T>, IListState<T>>
     }
 
     return (
-      <div ref={this._root} {...divProps} role={pageElements.length > 0 ? role : undefined} className={css('ms-List', className)}>
-        <div ref={this._surface} className={'ms-List-surface'} role="presentation">
-          {pageElements}
+      <>
+        <div ref={this._root} {...divProps} role={pageElements.length > 0 ? role : undefined} className={css('ms-List', className)}>
+          <div ref={this._surface} className={'ms-List-surface'} role="presentation">
+            {pageElements}
+          </div>
         </div>
-      </div>
+        <EventListener target={window} type="resize" listener={this._onAsyncResize} />
+        <EventListenerWithComputedTargetProps
+          capture
+          resolveTarget={this._getScrollableElement}
+          type="focus"
+          listener={this._onFocus.bind(this)}
+        />
+        <EventListenerWithComputedTargetProps
+          capture
+          resolveTarget={this._getScrollableElement}
+          type="scroll"
+          listener={this._onScroll.bind(this)}
+        />
+        <EventListenerWithComputedTargetProps
+          capture
+          resolveTarget={this._getScrollableElement}
+          type="scroll"
+          listener={this._onAsyncScroll.bind(this)}
+        />
+      </>
     );
   }
 
+  private _getScrollableElement = (): HTMLElement => findScrollableParent(this._root.current) as HTMLElement;
```

Comment:
how come this getter is now being to the finder's value after its usage in resolveTarget above?